python/pyfeelpp-toolboxes/feelpp/toolboxes/fluid/fluid.cpp

```diff
@@ -26,6 +26,9 @@
 #include <feel/feelpython/pybind11/functional.h>
 #include <feel/feelmodels/modelcore/modelnumerical.hpp>
 #include <feel/feelmodels/fluid/fluidmechanics.hpp>
+#include <feel/feelmodels/modelcore/remeshinterpolation.hpp>
+#include <feel/feelcore/pybind11_json.hpp>
+#include "contactforce.hpp"
```

Comment:
use < >

Reply:
Where can I put the file `contactforce.hpp` (contains the collision algorithm) ? in which directory ?

Reply:
I used `include_directories` in the CMakeList to include the` contactforce.hpp` file. I couldn't use `target_include_directories` with the macro `feelpp_add_pymodule`.